torch/_inductor/ir.py

```diff
@@ -7289,6 +7301,59 @@ def codegen(self, wrapper):
         wrapper.codegen_while_loop(self)
 
 
+class EffectfulKernel(FallbackKernel):
+    def __init__(
+        self,
+        layout,
+        kernel,
+        tensor_args,
+        nontensor_args,
+        unflatten_args,
+        kwargs=None,
+    ):
+        super().__init__(
```

Comment:
If in the future, if we are wrapping more than just single kernels with tokens, or if we decide to wrap non-Fallback kernels with tokens, we may need to rethink this.

But I think it's fine to land as is for now